src/bootstrap/main.py

```diff
@@ -65,13 +65,11 @@ def find_west_topdir(start):
 
     If none is found, raises WestNotFound.'''
     # If you change this function, make sure to update west.util.west_topdir().
-    def is_west_dir(d):
-        return os.path.isdir(d) and '.west_topdir' in os.listdir(d)
 
     cur_dir = os.getcwd()
 
     while True:
-        if is_west_dir(os.path.join(cur_dir, 'west')):
+        if os.path.isfile(os.path.join(cur_dir, WEST_DIR, WEST_TOPDIR)):
```

Comment:
Watch out when syncing this with https://github.com/zephyrproject-rtos/west/pull/40, since it removes `WEST_TOPDIR`.